consoleme/handlers/v2/resources.py

```diff
@@ -112,3 +116,79 @@ async def get(self, account_id, resource_type, region=None, resource_name=None):
                 config_timeline_url=resource_details.get("config_timeline_url"),
             )
         )
+
+
+class GetResourceURLHandler(BaseMtlsHandler):
+    """consoleme CLI resource URL handler. Parameters accepted: arn."""
+
+    def check_xsrf_cookie(self):
```

Comment:
I don't think we need check_xsrf_cookie here since we're not performing a PUT/POST.

Reply:
Oh cool, looks like I was able to push to this branch

Reply:
Yup, I checked the `Allow edits and access to secrets by maintainers` box when making the PR 😄